modules/data-entry/src/main/frontend/src/questionnaire/Subject.jsx

```diff
@@ -315,53 +320,81 @@ function SubjectMember (props) {
           : ""
         }
       </Grid>
-      <Grid item>
-      {tableData ?
-          (<Grid container spacing={3}>
-            {tableData.map( (entry, i) => {
-              return(
-                <Grid item lg={12} xl={6} key={`${entry.questionnaire["jcr:uuid"]}-${i}`}>
-                  <Card className={classes.subjectCard}>
-                    <CardHeader
-                      title={
-                        <React.Fragment>
-                          <Button size={buttonSize} className={classes.subjectFormHeaderButton}>
-                            {entry.questionnaire["@name"]}
-                          </Button>
-                          {entry["statusFlags"].map((status) => {
-                            return <Chip
-                              key={status}
-                              label={wordToTitleCase(status)}
-                              className={`${classes.subjectChip} ${classes[status + "Chip"] || classes.DefaultChip}`}
-                              size="small"
-                            />
-                          })}
-                        </React.Fragment>
-                      }
-                      className={classes.subjectFormHeader}
-                      action={
-                        <DeleteButton
-                          entryPath={entry["@path"]}
-                          entryName={`${identifier}: ${entry.questionnaire["@name"]}`}
-                          entryType="Form"
-                          warning={entry ? entry["@referenced"] : false}
-                        />
-                      }
-                    />
-                    <CardContent>
-                      <FormData formID={entry["@name"]} maxDisplayed={maxDisplayed}/>
-                      <Link to={"/content.html" + entry["@path"]}>
-                        <Typography variant="body2" component="p">See More...</Typography>
-                      </Link>
-                    </CardContent>
-                  </Card>
-                </Grid>
-              )
-            })}
-          </Grid>
-          ) : ""
+      { subjectGroups && Object.keys(subjectGroups).length > 0 && <Grid item>
+        <Grid container spacing={8}> {
```

Comment:
These should be on one column:
```suggestion
        <Grid container direction="column" spacing={8}> {
```